### PR TITLE
Passable bodies

### LINKED ISSUE #10: Passability is ignored
The information contained in `passable_bodies` is never propagated to any collider since we removed support for sensors.

### PR DESCRIPTION
**Pull request by tautropfli**
Resolves #10

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -26,3 +26,8 @@
 
 ## 0.8.0
 - Expose `objects_in_ray` through `WorldIterator::find_objects_in_ray`
+
+## TBD
+- `World::is_body_passable` has been removed. Use the `PhysicalBody::passable` (returned by `World::body`) instead.
```
Comment (janhohenheim) [inline changelog.md]: Remove the `the`

File: src/simulation/simulation_impl/world/nphysics_world.rs [hunk 2]
```diff
@@ -197,11 +223,18 @@ impl World for NphysicsWorld {
         /// Arbitrary value
         const MASS_OF_BODY_IN_KG: f64 = 20.0;
 
+        let collision_groups = if body.passable {
```
Comment (janhohenheim) [inline src/simulation/simulation_impl/world/nphysics_world.rs]: Can we move this into a function?

File: src/simulation/simulation_impl/world/nphysics_world.rs [hunk 3]
```diff
@@ -722,6 +799,103 @@ mod tests {
...
+        test_body_can_move_into_passable_body(
+            PhysicalBody {
+                location: Point { x: 15.0, y: 5.0 },
+                ..passable_body()
+            },
+            Point { x: 13.0, y: 5.0 },
+        );
+    }
+
+    fn test_body_can_move_into_passable_body(
```
Comment (janhohenheim) [inline src/simulation/simulation_impl/world/nphysics_world.rs:871]: remove the `test_` prefix

**Review by janhohenheim — CHANGES_REQUESTED** [review]

File: src/simulation/simulation_impl/world/nphysics_world.rs (hunk 3 above)
Comment (tautropfli) [reply src/simulation/simulation_impl/world/nphysics_world.rs:871]: This is intentional, as this is not directly a test function, but called from functions marked as `#[test]`

**Review by janhohenheim — DISMISSED** [review]

**Review by janhohenheim — APPROVED** [review]

**Pull request merged**